tools/cardano-sl-tools.cabal

```diff
@@ -363,8 +344,7 @@ executable cardano-keygen
 
   if flag(for-installer)
     buildable: False
-
-  if !flag(with-wallet)
+  else
     buildable: False
```

Comment:
This is wrong. I don't understand why there was `with-wallet` check. `buildable: False` should be set only if `for-installer` is enabled.